### PR TITLE
It looks better using ✔ instead of ✅

### PR DESCRIPTION
**Pull request by KingStudy**
It looks better using ✔ instead of ✅

### CODE REVIEW

**Comment by ycwei982** [comment]
i think this PR so funny HAHAHAHAHA

**Comment by lilydjwg** [comment]
screenshot

Really?

**Comment by ycwei982** [comment]
lilydjwg it just look better in Windows

**Comment by King66** [comment]
@ycwei982 your chinglish is very good.

**Comment by ycwei982** [comment]
@King66 well I don't mind too much grammar if I am not having an English test. lol

**Comment by KingStudy** [comment]
@ycwei982
夷陵的初中小朋友，好好读书去吧。
别当键盘侠了

File: README.md [hunk 1]
```diff
@@ -4,62 +4,44 @@
...
+| safari [🔊](http://dict.youdao.com/dictvoice?audio=safari&type=2) | ✔ [sə'fɑːrɪ] | ❌ [sæfərɪ] |
+| scheme [🔊](http://dict.youdao.com/dictvoice?audio=scheme&type=2) | ✔ [skiːm] | ❌ [s'kæmə] |
+| sudo | ✔ ['suːduː] | |
+| suite [🔊](http://dict.youdao.com/dictvoice?audio=suite&type=2) | ✔ [swiːt] | ❌ [sjuːt] |
+| typical [🔊](http://dict.youdao.com/dictvoice?audio=typical&type=2) | ✔ ['tɪpɪkl] | ❌ ['taɪpɪkəl] |
+| Ubuntu [🔊](http://dict.youdao.com/dictvoice?audio=ubuntu&type=2) | ✔ [ʊ'bʊntʊ] | ❌ [juː'bʊntʊ] |
+| width [🔊](http://dict.youdao.com/dictvoice?audio=width&type=2) | ✔ [wɪdθ] | ❌ [waɪdθ] |
+| YouTube [🔊](http://dict.youdao.com/dictvoice?audio=youtube&type=2) | ✔ ['juː'tjuːb] | ❌ ['juː'tʊbɪ] |
 
-
```
Comment (fghpdf) [inline README.md]: 是不是误删了参考资料以及我们团队的信息

✅   还是 ✔  不是这个项目的重点

**Comment by KingStudy** [comment]
@fghpdf 我第一次使用PR，测试一下功能，通过不通过都没什么的。

**Comment by fghpdf** [comment]
@KingStudy 好的

**Pull request closed without merging**